app/controllers/providers/means/capital_disregards/mandatory_controller.rb

```diff
@@ -10,7 +10,11 @@ def show
 
         def update
           @form = Providers::Means::CapitalDisregards::MandatoryForm.new(form_params)
-          render :show unless save_continue_or_draft(@form)
+
```

Comment:
You may be able to use this pattern

```ruby
ender :show unless save_continue_or_draft(@form, @legal_aid_application.next_incomplete_mandatory_disregard)
```

I believe any args are passed to go_forward anyway if form is valid

Reply:
You can namespace it with a key
```ruby
render :show unless save_continue_or_draft(@form, next: @legal_aid_application.next_incomplete_mandatory_disregard)
```
and then update the flow to expect arguments